<launch>

    <arg name="camera_topic"        default="/misty/id_0/camera/color/image_raw"/>


    <include file="$(find misty_wrapper)/launch/misty.launch" pass_all_args="true"></include>
    <node pkg="gaze_tracking_ros" type="gaze_tracker.py" name="gaze_tracker">
        <remap from="image_raw" to="$(arg camera_topic)" />
    </node>
    <node name="gaze_follower" pkg="robo_copilot" type="gaze_follower.py" output="screen"/>
</launch>